packages/browser/README.md

```diff
@@ -0,0 +1,170 @@
+var attachment2 = new BacktraceUint8ArrayAttachment("output.dmp", new Uint8Array(2));
+
+// Setup array of files to attach
+const attachments = [attachment1];
+
+// BacktraceClientOptions
+const options = {
+    // Name of the website/application
+    name: "MyWebPage",
+    // Version of the website
+    version: "1.2.3",  
+    // Submission url
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+    // Attach the files to all reports
+    attachments: attachments
+}
+
+// Later decide to add an attachment to all reports
+client.attachments.push(attachment2)
+
+// After catching an exception and generating a report
+try {
+    throw new Error("Caught exception!")
+} catch (error) {
+    const report = new BacktraceReport(error);
+    const attachment3 = new BacktraceStringAttachment("CaughtErrorLog", "some error logging data here");
+    report.attachments.push(attachment3);
+    client.send(report);
+}
+```
+
+### Metrics
+Backtrace Metrics tracks unique application launches and unique identifiers (default: guid) and submits them to Backtrace.  This allows for overview in the Backtrace web console of how many errors occur compared to all active users for a given platform, version, and more to be seen.
+
+Backtrace Metrics is enabled by default.  The following options can be passed to the `metrics` [BacktraceClientOptions](#backtraceclientoptions) item.
+
+| Option Name | Type | Description | Default |
+|-|-|-|-|
+|`metricsSubmissionUrl`|String|Metrics server hostname|`https://events.backtrace.io`|
+|`enable`|Boolean|Determines if the metrics support is enabled|`true`|
+|`autoSendInterval`|Integer|Indicates how often crash free metrics are sent to Backtrace. The interval is a value in ms. The default is 30 minutes|`1800000`|
+|`size`|Integer|Indicates how many events the metrics storage can store before auto submission|`50`|
+
+### Breadcrumbs
+Feature in developement
+
+### Callback Methods
+The Backtrace Browser has several callback methods that allow you to modify the reports
+#### BeforeSend
+Triggers an event every time an exception in the managed environment occurs, which allows you to skip the report (by returning a null value) or to modify data that library collected before sending the report. You can use the BeforeSend event to extend attributes or JSON object data based on data the application has at the time of exception or remove the same.
+
+```js
+import { BacktraceClient, BacktraceConfiguration, BacktraceData } from "@backtrace/browser";
+
+function myBeforeSend(data: BacktraceData): BacktraceData {
+    data.attributes.MyMessage = "Attribute testing!";
+    for(let key in data.attributes) {
+        console.log(`Key is: ${key} and value is ${data.attributes[key]}`);
+    }
+    return data;
+}
+
+const test: BacktraceConfiguration = {
+    name: "blah",
+    version: "1.2.3",
+    url: "https://submit.backtrace.io/<universe>/<token>/json",
+    beforeSend: myBeforeSend
+}
+```
```

Comment:
I would tentatively assert:
Callback methods would be part of Documentation.
"Features" would be info relating to how to support a Backtrace system/UI feature.
"Documentation" would be info on just the SDK - anything mechanical that happens prior to the report being sent.